modules/core/test/src/smithy4s/StubSmokeSpec.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s
+
+import munit._
+class DefaultServiceSmokeSpec() extends FunSuite {
+
+  test("Default stubs do compile") {
+    object stub extends smithy4s.example.Weather.Default[Option](None)
```

Comment:
Now that I think about it... why not make Default a trait with an abstract member?

Reply:
That's what I was thinking :)

Reply:
I naively tried to convert `Constant` to a `trait` but because it takes a parameter and that parameter is forwarded to the extends `Transformed` class, I could not pull it off

Reply:
@daddykotex is correct.

Reply:
ah, I see the issue now. had to try it by myself to understand :)

```scala
line"class $Constant_[P[-_, +_, +_, +_, +_]](value: P[Any, Nothing, Nothing, Nothing, Nothing]) extends $Transformed_[$opTraitNameRef, P](reified, $const5_(value))",
```